dracut/30ignition/coreos-populate-var.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Populate OSTree /var
 DefaultDependencies=false
+Before=initrd-switch-root.target
```

Comment:
Don't we only need this in the units that must be explicitly stopped on the way out?

Reply:
Yes, though I wanted to have a consistent "header" that's shared between all the units so we don't miss it when adding new units down the line (or adding `ExecStop=` to existing units).